hw1/02-palindrome.js

```diff
@@ -1,3 +1,30 @@
-const elem = document.querySelector('input');
+const inputElement = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+inputElement.addEventListener("input", isNumPalindrome);
+
+function isNumPalindrome() {
+  var num = document.getElementById("number").value;
+
+  result.textContent = "";
+
+  if (num === "") {
+    return;
+  }
+
+  if (num >= 0) {
+    var reversedNum = parseInt(num.toString().split("").reverse().join(""));
+
+    if (parseInt(num) === reversedNum) {
+      document.getElementById("result").textContent =
```

Comment:
It seems you're getting this document element quite a bit, consider storing it in a variable to use and lower the code duplication 👍